services.yaml

```diff
@@ -218,6 +261,37 @@ paths:
           $ref: "#/responses/OK"
         401:
           $ref: "#/responses/UnauthorizedError"
+  /services/{service_id}/team/change-role:
```

Comment:
this to me feels like we are changing the role of the team and not members of it.
i think a POST team/{member} is nicer.
If we are in need of batch here, than i would simply make the /team endpoint more sophisticated (and less restful) and have it as a general invite-or-change-member